src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -473,14 +497,14 @@ vector<SparqlTripleSimple> Visitor::visit(Parser::QuadDataContext* ctx) {
 vector<SparqlTripleSimple> Visitor::visit(Parser::QuadsContext* ctx) {
   if (!ctx->quadsNotTriples().empty()) {
     // Could also be default; disallow completely for now.
-    reportNotSupported(ctx->quadsNotTriples(0),
-                       "Named graphs are not supported.");
+    reportNotSupported(ctx->quadsNotTriples(0), "Named graphs are");
   }
 
   AD_CORRECTNESS_CHECK(ctx->triplesTemplate().size() == 1);
 
   auto convertAndRegisterTriple =
       [this](const std::array<GraphTerm, 3>& triple) -> SparqlTripleSimple {
+    // TODO:
```

Comment:
TOOD what exactly?

Reply:
Correctly do the variable registration. There is also an open review comment from the previous review for that.